github-pullrequest-plugin/src/main/resources/org/jenkinsci/plugins/github/pullrequest/events/impl/GitHubPRLabelAddedEvent/help.html

```diff
@@ -1,4 +1,5 @@
 <div>
-    Trigger when label was added to PR. When multiple label expected,
-    then job triggered only when all labels will appear in summary (in comparison to previous state).
+    Triggers a build when the label or labels specified are added to a
+    pull request. Note that with multiple labels, all of the labels must be
+    present to cause the trigger to fire.
```

Comment:
Triggering logic depends on 2 kind of actions: skip or trigger. For all possible label cases (added, existed, removed, contains) there are triggers. You can group them or order in UI according to waterfall to achieve some triggering action.